Makefile

```diff
@@ -23,6 +23,12 @@ LDFLAGS       := -ldflags "-X ${PKG_LDFLAGS}.Version=${VERSION} -X ${PKG_LDFLAGS
 
 all: clean build
 
+.PHONY: release
+release:
+	./ui_release.sh --release $(version)
+	cd ui/ && rm package-lock.json && npm install && cd ../
```

Comment:
yeah that's what I think too. So why on earth I did it ? That's because if I don't, then the command `npm install` is trying to download the new version which is weird.
And when I'm deleting the lock file, then it creates it with the correct version, and it doesn't change / update the other dependencies. Which is also weird.
So I have no idea if I should remove this part of the command or not.

Reply:
It appears it was a bug in npm. Upgrading to npm `v8.5.0` solves the issue. So no need anymore to remove the file `package-lock.json`